docs/templating-guide/protocols/http.md

```diff
@@ -579,6 +579,7 @@ Request inline annotations allow performing per request properties/behavior over
 
 - `@Host:` which overrides the real target of the request (usually the host/ip provided as input). It supports syntax with ip/domain, port, and scheme, for example: `domain.tld`, `domain.tld:port`, `http://domain.tld:port`
 - `@tls-sni:` which overrides the SNI Name of the TLS request (usually the hostname provided as input). It supports any literals, the speciale value `request.host` use the value of the `Host` header.
+- `@timeout:` which overrides the timeout for the request to a custom duration. It supports durations formatted as string. If no duration is specified, the default Timeout flag value is used.
```

Comment:
What do you mean by formatted as string?

Is the "seconds" marked by an "s" after a number is the only format supported, or can you also specify minutes or milliseconds? It would worth mentioning this...